docs/release-notes.md

```diff
@@ -9,20 +9,107 @@ description: >
 We recommend always using "tagged" versions of Ondat rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-The latest tagged release is `2.8.0`. For
+The latest tagged release is `2.8.1`. For
 installation instructions see our
 [Install](/docs/reference/cluster-operator/install) page.
 
-The latest CLI release is `2.8.0`, available from
+The latest CLI release is `2.8.1`, available from
 [GitHub](https://github.com/storageos/go-cli/releases).
 
 # Upgrading
 
 To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.
 
-## 2.8.0 - Release 2022-06-29
+## 2.8.1 - Release 2022-08-02
+
+### New
+
+k8s
+
+* Install resource quota for GKE
+* Disable scheduler option in storageoscluster CR
+* Operator:
+  * Add snapshot controller and related CRDs if not present
+  * Pod Disruption Budget support for k8s v1.25
+* Improve logging on kubectl plugin <https://github.com/storageos/kubectl-storageos/pull/194>
+
+Control Plane
+
+* Adds randomisation behaviour of placement
+
+Data Plane
 
-2.8.0 Release Notes
+* Warn that filesystem might go read-only after a failed write/unmap/sync SCSI command. The log of interest is: `"SCSI command failed - if the block device is mounted the filesystem may go read-only".`
+* Log when the average IO service time from the mount node is greater than 2 seconds. We log the following message on a per volume basis with exponential backoff: `"it is taking unsually long to send and receive IO from the presentation node"`. Metrics are included in the log message.
+  * Note: this measurement is tracking the time it takes to send the IO over the network to the master and any replicas and for the IO to be committed and the response sent back to the mount node.
+* Log when the average IO service time from the master node to its replica is greater than 2 seconds. We log the following message, on a per replica basis with exponential backoff: "it is taking unsually long to send and receive IO from the master deployment to its replica". Metrics are included in the log message. Note: this measurement is tracking the time it takes to send the IO over the network to the replica and for the IO to be committed and the response sent back to the mount node.
+* Log when it takes more than 1 second to commit a write, read, sync or unmap to disk. Logs of interest are of the format `"X operation took longer than Yms to complete completion_time=Zms".`
+* Adds support for per volume IO tracing. For each mounted volume we'll dump the following every 60 seconds
```

Comment:
Tracing is a hidden feature, so shouldn't be in the user-facing release notes!

Reply:
@geordieintheshellcode done